newIDE/app/src/Utils/GDevelopServices/Tutorial.js

```diff
@@ -2,14 +2,14 @@
 import axios from 'axios';
 import { GDevelopAssetApi } from './ApiConfigs';
 
-export type Tutorial = {
+export type Tutorial = {|
```

Comment:
Note that in theory we can't guarantee that the backend won't one day return an additional key.

Reply:
Yes of course, but we should tell Flow that we are confident the type will be strictly this. I think it's a different issue that Flow cannot do anything about